dataplicity/m2m/wsclient.py

```diff
@@ -188,8 +191,7 @@ def opened(self):
     def received_message(self, message):
         """Call on_message with binary packets."""
         try:
-            if message.is_binary:
-                self.on_message(self, message.data)
+            self.on_message(self, message)
```

Comment:
We're going to pass on ws4py's messages. Not all messages may be binary packets, so `on_message` has been updated accordingly.